docs-chef-io/content/inspec/cli.md

```diff
@@ -1,17 +1,10 @@
 +++
-title = "InSpec CLI"
-draft = false
-gh_repo = "inspec"
-
-[menu]
-  [menu.inspec]
-    title = "InSpec Executable"
-    identifier = "inspec/reference/cli.md InSpec Executable"
-    parent = "inspec/reference"
-    weight = 10
+title: "InSpec CLI"
+draft: false
+gh_repo: "inspec"
 +++
 
-<!-- markdownlint-disable MD024 -->
+# InSpec CLI
```

Comment:
We don't need this. It's defined in the page frontmatter

Reply:
Okay, will remove this.